src/server/standalone.js

```diff
@@ -642,9 +642,7 @@ function StandAloneServer(gmeConfig) {
     // Basic authentication
 
     logger.debug('creating login routing rules for the static server');
-    __app.get('/', ensureAuthenticated, function (req, res) {
-        expressFileSending(res, __clientBaseDir + '/index.html');
-    });
+    __app.get('/', ensureAuthenticated, Express.static(__clientBaseDir));
```

Comment:
I do not think this is right... does the UI load for you? Visiting only http://localhost:8888/ ? Or express sends the index.html by default?